docs/certbot_authenticators.md

```diff
@@ -0,0 +1,92 @@
+
+
+## Setting up the container to use DNS-01 challenges
+
+To use DNS-01 challenges, you will need to create the credentials file for the chosen authenticator.
+
+You can find information about how to configure them by following those links for the supported authenticators:
+ - [dns-cloudflare][6]
+ - [dns-cloudxns][7]
+ - [dns-digitalocean][8]
+ - [dns-dnsimple][9]
+ - [dns-dnsmadeeasy][10]
+ - [dns-gehirn][11]
+ - [dns-google][12]
+ - [dns-linode][13]
+ - [dns-luadns][14]
+ - [dns-nsone][15]
+ - [dns-ovh][16]
+ - [dns-rfc2136][17]
+ - [dns-route53][18]
+ - [dns-sakuracloud][19]
+
+You will need to setup the authenticator file at `/etc/letsencrypt/<authenticator provider>.ini`.
+E.g. for Cloudflare, we would create the file `/etc/letsencrypt/cloudflare.ini` with the following content:
+
+```ini
+# Cloudflare API token used by Certbot
+dns_cloudflare_api_token = 0123456789abcdef0123456789abcdef01234567
+```
+
+
+## Using a DNS-01 authenticator by default
+
+You can use an authenticator solving DNS-01 challenges by default by setting the `CERTBOT_AUTHENTICATOR` environment variable in your docker configuration with for value the name of the authenticator you wish to use (e.g. `dns-cloudflare`).
+
+All the certificates needing renewal or creation will then start using that authenticator. Make sure, of course, that you've setup the authenticator correctly, as described above.
+
+
+## Using a DNS-01 authenticator for specific certificates only
+
+You might want to keep using the `webroot` authenticator in most cases, but need to use a DNS-01 challenge to setup a wildcard certificate for a given domain. Or you might even have a domain setup on Route53 while your other domains are on Cloudflare, and you thus are using `dns-cloudflare` as default authenticator.
+
+In such cases, you can specify the authenticator you wish to use in the certificate path that you are setting up as `ssl_certificate_key` in your server block of the nginx configuration. In our case, if we want to use `dns-route53` for a specific certificate, we could be using the following:
+
+```
+server {
+    listen              443 ssl;
+    server_name         yourdomain.org *.yourdomain.org;
+    ssl_certificate_key /etc/letsencrypt/live/test-name.dns-route53/privkey.pem;
+    ...
+}
+```
+
+The script ran in the container to renew certificates will automatically identify that it needs to use the Route53 authenticator here. Of course, you will need that authenticator to be configured properly in order to be able to use it.
```

Comment:
Add a note here stating that it will work with the RSA/ECC configuration at the same time? 
https://github.com/JonasAlfredsson/docker-nginx-certbot/blob/master/docs/advanced_usage.md#multi-certificate-setup